src/material/chips/chip-input.ts

```diff
@@ -125,6 +129,12 @@ export class MatChipInput implements MatChipTextControl, OnChanges, OnDestroy {
   /** The native input element to which this directive is attached. */
   readonly inputElement!: HTMLInputElement;
 
+  /**
+   * Default Screen-reader placeholder for the input if no placeholder or
+   * ariaPlaceholder is provided.
+   */
+  private readonly _defaultAriaPlaceholder = 'Enter input';
```

Comment:
I don't think we should be providing a default here, but advise users that they should set it themselves.